docs/index.rst

```diff
@@ -72,7 +72,7 @@ The configuration parameters that can be set on this page are divided into a set
         | |sparkplug_4.1| |
         +-----------------+
 
-        - *Asset Name*: Fixed asset name.
+        - *Asset Name*: Fixed asset name. Asset name should not be blank or whitespace; will be replaced by subscribed topic if left blank.
```

Comment:
If the asset name is left empty or only contains whitespace then the topic of the incoming MQTT message will be used as the asset name.